### PR TITLE
Update Feature Request and PR templates

### PR DESCRIPTION
**Pull request by opusforlife2**
#### What is it?
- [ ] Bug fix
- [ ] Feature
- [x] Meta improvement to project

#### Long description of the changes in your PR

- Minor alterations to the PR template
- Made the Feature Request template easier to fill out

#### Agreement
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -7,22 +7,33 @@ assignees: ''
 ---
 <!-- Hey. Our contribution guidelines (https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) might be an appropriate
  document to read before you fill out the request :) -->
-#### Is your feature request related to a problem? Please describe it
```
Comment (TobiGr) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Related issues and PRs should be mentioned somewhere

Comment (opusforlife2) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ah. Point.

Comment (opusforlife2) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Okay now?

**Comment by opusforlife2** [comment]
@TobiGr could you merge this soon? People are frequently ignoring the template, which I hope this PR will reduce incidences of.

**Pull request merged**